src/lib/blockchain_snark/blockchain_snark_state.ml

```diff
@@ -431,7 +431,7 @@ end) : S = struct
 
   let tag, cache_handle, p, Pickles.Provers.[ step ] =
     Pickles.compile () ~cache:Cache_dir.cache ~public_input:(Input typ)
-      ~auxiliary_typ:Typ.unit
+      ~auxiliary_typ:Typ.unit ~override_wrap_domain:N1
```

Comment:
isn't this an issue though, as this can reduce the size of all zkapps?

Reply:
What's actually happening here is: some circuits with '2 previous proofs' in the recursion now have a smaller domain size, so we need to override that domain size to make things work again. This flag gives us the ability to make that override.

Some zkApps with a small number of inductive rules and 2 previous proofs will also experience this, so we will need to expose the flag to SnarkyJS too.